Docs/upgrade.md

```diff
@@ -1,6 +1,25 @@
 # Upgrade LoRaWAN to a new version
 <!-- allow github to generate 'copy' buttons for coded URLs -->
 <!-- markdownlint-disable MD040 -->
+## Release 1.0.7
+
+> Note that this version switch IoT Edge to use version 1.2 of IoT Edge that include major changes on the edge. Please follow this [documentation](https://docs.microsoft.com/azure/iot-edge/how-to-update-iot-edge?view=iotedge-2020-11&tabs=linux) to upgrade IoT Edge to 1.2.
```

Comment:
nit: not sure how you do this usually, but the official name would be Azure IoT Edge.